decred/decred/dcr/account.py

```diff
@@ -1483,6 +1653,7 @@ def purchaseTickets(self, qty, price):
                 self.addMempoolTx(tx)
         # Store the txids.
         self.tickets.extend([tx.txid() for tx in txs[1]])
+        self.updateSpentTickets()
```

Comment:
Now checking dcrdata for new tickets happens in sync and is stored in txs. Also, added an extra check so that we don't fetch data for known unspent tickets. The only extra data that we can't get from the utxo in utxos is the initial transaction fee. However, that is only a few sats, and will be recalculated once the ticket is spent anyway, we could skip fetching that for live tickets and then remove one fetch for a tx on each new live ticket. As the fee is near zero, I think it would be ok. Actually, it depends on the size of the transaction/script, which we do have, so... wait a minute.

Reply:
Using an estimated fee for purchases now. The footprint should be be much smaller now, and this operations is only saving the utxo in utxos to the database and calling unspent tickets signal.

Reply:
> The only extra data that we can't get from the utxo in utxos is the initial transaction fee. 

> Using an estimated fee for purchases now. 

We have the transaction though, don't we? How can we not know the transaction fee?